dex/src/critbit.rs

```diff
@@ -252,13 +251,13 @@ const_assert_eq!(_NODE_ALIGN, align_of::<AnyNode>());
 
 #[derive(Copy, Clone)]
 #[repr(packed)]
-struct SlabHeader {
-    bump_index: u64,
-    free_list_len: u64,
-    free_list_head: u32,
-
-    root_node: u32,
-    leaf_count: u64,
+pub struct SlabHeader {
+    pub bump_index: u64,
+    pub free_list_len: u64,
+    pub free_list_head: u32,
+ 
```

Comment:
Format :)

Reply:
pretty sure my most recent commit fixed the formatting issue, let me know if it doesnt.